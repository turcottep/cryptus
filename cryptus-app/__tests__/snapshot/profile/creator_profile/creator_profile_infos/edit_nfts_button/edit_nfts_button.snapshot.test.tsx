// components/__tests__/button.snapshot.test.tsx
import React from "react";
import renderer from "react-test-renderer";
import EditNFTsButton from "../../../../../../components/profile/creator_profile/creator_profile_infos/edit_nfts_button/edit_nfts_button";

it("EditNFTsButton renders correctly", () => {
  const tree = renderer.create(<EditNFTsButton username={"tester"} />).toJSON();
  expect(tree).toMatchSnapshot();
});
