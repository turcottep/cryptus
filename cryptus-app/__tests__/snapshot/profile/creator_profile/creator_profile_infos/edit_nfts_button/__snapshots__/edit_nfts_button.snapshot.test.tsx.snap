// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditNFTsButton renders correctly 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    Edit NFTs
  </button>
</div>
`;
